packages/zoe/src/contracts/priceAggregator.js

```diff
@@ -62,8 +63,9 @@ const priceDescriptionFromQuote = quote => quote.quoteAmount.value[0];
  * quoteMint?: ERef<Mint<'set'>>,
  * storageNode: ERef<StorageNode>,
  * }} privateArgs
+ * @param {import('@agoric/vat-data').Baggage} baggage
  */
-const start = async (zcf, privateArgs) => {
+const start = async (zcf, privateArgs, baggage) => {
```

Comment:
please leave this contract alone because it's just an example, not part of production
https://github.com/Agoric/agoric-sdk/blob/bb30947461eae614ff5e4fac3a9e21b927efe3f6/packages/zoe/src/contracts/priceAggregator.js#L47-L52

It is confusing that `zoe/src/contracts` has examples and production contracts. We have deferred cleaning up package factoring https://github.com/Agoric/agoric-sdk/issues/4645

Reply:
also:
 - #1656

Reply:
Reverted, and annotated contract to be `notForProductionUse`